src/pages/index.tsx

```diff
@@ -1,38 +1,35 @@
 import styled from "@emotion/styled";
 import Head from "next/head";
 
-import {
-  Header,
-  StartPage,
-  MockupPageProject,
-  MockupPagePortfolio,
-  AboutPage,
-  LastPage,
-} from "Components";
+import { Header, StartPage } from "Components";
 import React from "react";
+import path from "constants/path.json";
 
 const Wrapper = styled.div`
   background: url(/StartBackground.svg);
 `;
 
-const index: React.FC = () => {
+// TODO login token 에 따라 index page 가 바뀌어야 함
+// TODO token ? searchResultPage : StartPage
+
+const Index: React.FC = () => {
   return (
     <div>
       <Head>
         <title>Olio</title>
         <link rel="icon" href="/favicon.ico" />
+        <link rel="preconnect" href={path.font_gstatic} />
+        <link href={path.josefin_font} rel="stylesheet" />
```

Comment:
It would be good to unify the order of the props of the link.
```suggestion
        <link rel="stylesheet" href={path.josefin_font} />
```